src/polymer-lint.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Is this file meant to be checked in, or is it just included to show us how warnings can be interacted with?